bare/roles/web/tasks/letsencrypt.yml

```diff
@@ -19,7 +19,11 @@
   command: "systemctl reload nginx"
 
 - name: Install letsencrypt cert
-  command: letsencrypt certonly -n --webroot -d {{ mastodon_host }} -w {{ mastodon_home }}/{{ mastodon_path }}/public/ --email "webmaster@{{ mastodon_host }}" --agree-tos && systemctl reload nginx
+  command: letsencrypt certonly -n --webroot -d {{ mastodon_host }} -w {{ mastodon_home }}/{{ mastodon_path }}/public/ --email "webmaster@{{ mastodon_host }}" --agree-tos
+  when: not letsencrypt_cert.stat.exists
+
+- name: Reload nginx
+  command: "systemctl reload nginx"
   when: not letsencrypt_cert.stat.exists
```

Comment:
Not sure whether I understand you correctly:

The syntax before could not have work because it used shell syntax in a call to the `command` module.

And in the suggested change, nginx will only be reloaded when the certbot command has been succeful, in case of failure ansible should abort (no ignore_errors here or so). 

Maybe I misunderstood?

Reply:
Fair, I meant to use a different syntax, but I think this is sufficient